packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.tsx

```diff
@@ -12,7 +12,7 @@ const renderItemIcon = (props: IContextualMenuItemProps) => {
   } = props;
 
   // Only present to allow continued use of item.icon which is deprecated.
```

Comment:
Nit: this comment doesn't seem accurate anymore since there is no more continued use of item.icon.